src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -1,145 +1,188 @@
-            <Private>True</Private>
-        </Reference>
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\AttributeFilter\AttributeFilter.csproj" />
+    <ProjectReference Include="..\Configuration\Configuration.csproj" />
+    <ProjectReference Include="..\Extensions\NewRelic.Agent.Extensions\NewRelic.Agent.Extensions.csproj" />
+    <ProjectReference Include="..\LabelsService\LabelsService.csproj" />
+    <ProjectReference Include="..\NewRelic.Core\NewRelic.Core.csproj" />
+    <ProjectReference Include="..\Parsing\Parsing.csproj" />
+  </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-        <Reference Include="Dongle">
-            <HintPath>$(SolutionDir)ExternalLibs\netstandard2.0\dongle.dll</HintPath>
-            <Private>True</Private>
-        </Reference>
-    </ItemGroup>
+  <PropertyGroup>
+    <ConfigurationCsPath>Config\Configuration.cs</ConfigurationCsPath>
+    <ExtensionCsPath>NewRelic.Agent.Core.Extension\Extension.cs</ExtensionCsPath>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <Reference Include="ICSharpCode.SharpZipLib">
-            <HintPath>$(SolutionDir)ExternalLibs\ICSharpCode.SharpZipLib.dll</HintPath>
-            <Private>True</Private>
-        </Reference>
-    </ItemGroup>
-    <ItemGroup>
-        <ProjectReference Include="..\AttributeFilter\AttributeFilter.csproj" />
-        <ProjectReference Include="..\Configuration\Configuration.csproj" />
-        <ProjectReference Include="..\Extensions\NewRelic.Agent.Extensions\NewRelic.Agent.Extensions.csproj" />
-        <ProjectReference Include="..\LabelsService\LabelsService.csproj" />
-        <ProjectReference Include="..\NewRelic.Core\NewRelic.Core.csproj" />
-        <ProjectReference Include="..\Parsing\Parsing.csproj" />
-    </ItemGroup>
-    <ItemGroup>
-        <Compile Include="..\..\..\Shared\SharedAssemblyInfo.cs">
-          <Link>Properties\SharedAssemblyInfo.cs</Link>
-        </Compile>
-        <Compile Include="$(SolutionDir)Shared\SharedLog4NetRepository.cs">
-            <Link>Properties\SharedLog4NetRepository.cs</Link>
-        </Compile>
-        <Compile Update="AgentInstallConfiguration_Generated.cs">
-          <DesignTime>True</DesignTime>
-          <AutoGen>True</AutoGen>
-          <DependentUpon>AgentInstallConfiguration_Generated.tt</DependentUpon>
-        </Compile>
-        <Compile Update="Properties\Resources.Designer.cs">
-            <AutoGen>True</AutoGen>
-            <DesignTime>True</DesignTime>
-            <DependentUpon>Resources.resx</DependentUpon>
-        </Compile>
-
-        <!-- The following includes are required because these files are generated by the PreBuildEvent. They do not exist when building from a clean working folder. 
+  <ItemGroup>
+    <Compile Include="..\..\..\Shared\SharedAssemblyInfo.cs">
+      <Link>Properties\SharedAssemblyInfo.cs</Link>
+    </Compile>
+    <Compile Include="$(SolutionDir)Shared\SharedLog4NetRepository.cs">
```

Comment:
Shared is in src\Agent

Reply:
This is correct, until the FullAgent.sln is moved.